### PR TITLE
train & eval scripts

### PR DESCRIPTION
**Pull request by lannelin**

### CODE REVIEW

**Comment by lannelin** [comment]
Hi @eddableheath  @J-Dymond . I've run this locally for 2 epochs to train a model which gets `zero one acc: 0.23 hamming acc: 0.92` on validation set (though not with seed set :( . )
Let me know if you'd like to review and then run on baskerville - possibly with slightly tweaked hparams - or prefer me/one of you to run this on baskerville first.

**Review by J-Dymond — COMMENTED** [review]
Looks good! I think it would be a good idea to train it for longer on Baskerville. This performs better than the zero-shot, as I'm sure we expected,  so I can work to get this into the current pipelines. We also have the more difficult levels of topic label which we can use, though we could probably move to training on those labels that at a later stage?

**Comment by lannelin** [comment]
closes #23

**Review by J-Dymond — APPROVED** [review]
These changes look reasonable, will merge them into inference pipeline branch before I open a pull request.
